### PR TITLE
[Feature][Connector-V2] Support multi-table sink feature for kudu

### LINKED ISSUE #5719: [Feature][Connector-V2] Support multi-table sink feature for kudu
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Support multi-table sink feature for kudu

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Carl-Zhou-CN**
### Does this PR introduce _any_ user-facing change?

close #5719 

### How was this patch tested?
new  e2e test

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
Please waiting #5955 be merged, then do test again.
image

**Comment by Carl-Zhou-CN** [comment]
> Please waiting #5955 be merged, then do test again. image

ok

File: docs/en/connector-v2/sink/Kudu.md [hunk 1]
```diff
@@ -123,6 +123,79 @@ sink {
...
 ```
 
+### Multiple Table
+
+```hocon
+env {
+  # You can set engine configuration here
+  execution.parallelism = 1
+  job.mode = "BATCH"
+  execution.checkpoint.interval = 5000
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Kudu.md]: Remove `execution.checkpoint.interval = 5000`

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kudu-e2e/src/test/java/org/apache/seatunnel/e2e/connector/kudu/KuduWIthMultipleTableIT.java [hunk 2]
```diff
@@ -210,10 +215,6 @@ private void initializeKuduTable() throws KuduException {
                 new ColumnSchema.ColumnSchemaBuilder("val_unixtime_micros", Type.UNIXTIME_MICROS)
                         .nullable(true)
                         .build());
-        columns.add(
```
Comment (EricJoy2048) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kudu-e2e/src/test/java/org/apache/seatunnel/e2e/connector/kudu/KuduWIthMultipleTableIT.java:213]: Why remove `binary` type?

File: docs/en/connector-v2/sink/Kudu.md (hunk 1 above)
Comment (Carl-Zhou-CN) [reply docs/en/connector-v2/sink/Kudu.md]: done

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kudu-e2e/src/test/java/org/apache/seatunnel/e2e/connector/kudu/KuduWIthMultipleTableIT.java (hunk 2 above)
Comment (Carl-Zhou-CN) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kudu-e2e/src/test/java/org/apache/seatunnel/e2e/connector/kudu/KuduWIthMultipleTableIT.java:213]: binary type, difficult to generate in fake, and difficult to compare,KuduIT contains all types of checks

**Review by Hisoka-X — APPROVED** [review]
LGTM

**Review by hailin0 — APPROVED** [review]

**Pull request merged**